src/app/app.component.html

```diff
@@ -35,12 +36,12 @@ <h1 class="example-app-name">MIFOS</h1>
-                       <p>  Welcome, Smith R </p>
+                       <p class="font-style" data-aos="fade-up">  Welcome, Smith R </p>
```

Comment:
@CajetanRodrigues Why have you hardcoded this? if you are fetching client details then put his/her name.

Reply:
Ok I will do that

Reply:
Done